spec/system/candidate_interface/entering_details/degrees/new_degree_flow/candidate_editing_degrees_spec.rb

```diff
@@ -75,6 +75,15 @@
     then_i_see_another_masters_degree_selected
   end
 
+  scenario 'editing international degree' do
```

Comment:
maybe we should move this scenario to it's own system spec to avoid multiple scenarios in one? 🤔

Reply:
yep, i can do that 👍

Reply:
I've added an international degree system spec